doc/content/microreactors/airjacket/index.md

```diff
@@ -0,0 +1,502 @@
+`usrdat2`). We create element maps/masks, that store local element and face IDs based on the element's centroid being between a certain
+coordinate range, and the face lying along a normal
+
+!listing microreactors/gcmr/airjacket/airjacket.usr start=element map of lower plane end=test_emap include-end=True
+
+The above code will store zeros in the `emap_lo` array, indicating the local element ID corresponding to that index of `emap_lo` belongs
+to an element whose centroid does not lie in the specified range of x and y values. For elements that do lie within the specified coordinate
+range, the corresponding value of `emap_lo` at the index corresponding to the local element ID stores the face that has a normal closest to the
+normal specified. To verify the correct elements and faces have been selected, the `test_emap` function can be used. It will create a Nek output
+file with the name `elo<casename>.nek5000`, which can be opened in Paraview. Look at the `x-velocity` field - the element map `emap_lo` will be
+highlighted with values equal to 1.0 at each node on the face. The rest of the points will be zero.
+
+Now that we have stored the element and face IDs that we need, we can average over faces using functions in the `aravg.f` file, that rely on Nek5000's
+internal functions `facint_v` and `facint_a`, which require element and face IDs as arguments. This will allow us to calculate area averages across
+the selected faces. Volumetric averages can be computed by constructing masks of nodes between a range of x and y values, using the following code
+
+!listing microreactors/gcmr/airjacket/airjacket.usr start=create masks end=create_mask include-end=True
+
+These element maps can be stored in a `common` block, so that they are accessible in `userchk` despite being created in `usrdat2`. To average any
+of the Nek5000 solution arrays over these element maps, simply call the following functions from `aravg.f` in `userchk`, such as
+
+!listing microreactors/gcmr/airjacket/airjacket.usr start=calculating averages end=tauinav include-end=True
+
+Bulk averages can be calculated similarly using the point masks and the `bulkavg` function. Weighted scalar averages or area integrals can be
+calculated using the `emap_scalar_avg` function. The variables can be saved, for example, using a common block, and the difference between
+their values at successive time-steps printed to monitor convergence. Gradients for calculating the Nusselt number are obtained using Nek5000's
+`gradm1` subroutine, and the gradients are averaged over the hot-wall to obtain a heat flux using Nek5000's `facint` function.
+
+
+### Input parameters
+
+A list of miscellaneous input parameters is included in [input_parameters] for reproducibility.
+
+!table id=input_parameters caption=Simulation input parameters.
+| Parameter  | Value  | Unit  |
+| :- | :- | :- |
+| Inlet temperature | 323.15 | K |
+| Outlet temperature | 693.15 | K |
```

Comment:
why is this outlet temperature so high?

Reply:
Thanks for catching that... that should say "hot wall temperature", outlet temperature is not available as an input param